recipes/_debian_galera.rb

```diff
@@ -35,7 +35,12 @@
   end
 end
 
-template '/var/cache/local/preseeding/mariadb-galera-server.seed' do
+preseed_file = '/var/cache/local/preseeding/mariadb-galera-server.seed'
```

Comment:
I'd prefer a if/else block here. As it's clearer, rather than overriding the value